packages/contracts/src/yield/LiquidContinuousMultiTokenVault.sol

```diff
@@ -274,6 +274,17 @@ contract LiquidContinuousMultiTokenVault is
         super.setReducedRate(reducedRateScaled_, effectiveFromPeriod_);
     }
 
+    /**
+     * @dev Withdraws the assets from the vault to the custodian.
+     * Only the Operator can call this function.
+     *
+     * @param to The address that will receive the assets. Custodian wallet.
+     * @param amount The amount of the ERC-20 underlying assets to be withdrawn from the vault.
+     */
+    function withdrawAsset(address to, uint256 amount) public onlyRole(DEFAULT_ADMIN_ROLE) {
+        _withdrawAssest(to, amount);
+    }
```

Comment:
recommend we create a new Access Control role specifically for this. something like:
- INVESTMENT_MANAGER